lib/lita-lean_tc.rb

```diff
@@ -62,6 +66,15 @@ def build_finished(payload)
           create_confirmed
         end
       end
+      # Returns a list of all cards on the Feature request board
+      #
+      def list_feature_request(response)
```

Comment:
I'm not entirely sure I'm able to use the `to_msg` method here as the end result is different to what I'm expecting.
This method is responding to a route, triggered by a user at any time in slack, and returns all the feature request cards. In `feature_requests.rb` the `to_msg` method is only returning a select few cards depending on their first(last) action date. Instead of responding to a user command, it's triggered by the timer. 
I'm not sure how to extract the board information out to allow it to be used for both a lita response and a timer.